src/manipulation.js

```diff
@@ -17,7 +17,7 @@ define([
 
 var
 	rxhtmlTag = /<(?!area|br|col|embed|hr|img|input|link|meta|param)(([\w:]+)[^>]*)\/>/gi,
-	rtagName = /<([\w:]+)/,
+	rtagName = /<([\w:-]+)/,
```

Comment:
Thanks @gibson042. As soon as I won the fight with `git rebase`, I'll push the update.